.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Enhancement
+about: Improve an existing feature or workflow
+labels: ['kind/enhancement', 'status/triage']
+---
+
+### What do you want to improve?
+
+### What is the current behavior?
+
+### What is the new behavior?
```

Comment:
```suggestion
### Expected behavior?
```

Reply:
I think it makes more sense to explicitly ask for the "new behavior" rather than an expected behavior since this is used for requesting a new feature or improvement.

Reply:
okay, let's make it future tense. like `How would you like the new behavior to be?` or some thing similar